example/create_dataset/main.tf

```diff
@@ -4,23 +4,28 @@ terraform {
       source  = "hashicorp/google"
       version = "~> 4.13.0"
     }
-    time = {
-      source  = "hashicorp/time"
-      version = "0.7.2"
-    }
     random = {
       source  = "hashicorp/random"
       version = "3.1.2"
     }
   }
 }
 
+resource "random_id" "random_id" {
+  byte_length = 4
+}
+
+resource "google_service_account" "google_service_account" {
```

Comment:
this name can definitely be improved. The type is already `google_service_account` the name should be something more descriptive.

Reply:
I would suggest `"google_service_account" "dataset_owner_account"`

Reply:
Fixed to `owner`